onnx/mapping.py

```diff
@@ -97,7 +97,7 @@ def __getitem__(self, key: Union[int, str, np.dtype]) -> Any:
             warnings.warn(
                 str(
```

Comment:
Remove str and do

```python
    f"`mapping.{self._origin_function}` is now deprecated and will be removed in the next release or so."
    f"To silence this warning, please use `helper.{self._future_function}` instead."
```

?

optional, nit:

"in the next release or so" -> "in a future release"?